INVENTORY.md

```diff
@@ -0,0 +1,22 @@
+# Inventory
+
+this page talks about common inventory information information provided by DPU vendors in a same format
+
+- Questions:
+  - Where does it run?
```

Comment:
Does this matter? I would imagine each [D,I]PU vendor may run things slightly differently. I think what we should be focusing on here is the inventory information itself, rather than how it is collected

Reply:
it matters because in different runtime environments maybe we provide different information (less or more) like in UEFI vs Linux we can have 2 versions of packets or something like that.. hope it make sense...

Reply:
depends on the use case... if some specific provisioning system need to have information that UEFI collector can't provide, you need to boot into Linux to get this more comprehensive information to proceed... 
this is open question.... I don't know... I'm saying there is a possibility that we have to explore

Reply:
This starts to feel to me like we're coming dangerously close to defining *how [D,I]PUs are built* rather than defining what information they emit. But I agree, exploration here is important and can help to quell some of my fears.

Reply:
My point simple... should we define more then 1 inventory packet version or not... 

all the rest are examples and supporting material why should we...